sdk/core/azure-core/src/main/java/com/azure/core/util/polling/PollerFlux.java

```diff
@@ -162,6 +162,22 @@ private PollerFlux(Duration pollInterval,
         this.syncActivationOperation = cxt -> activationOperation.apply(cxt).block();
     }
 
+    /**
+     * Creates a poller flux instance that returns an error on subscription.
```

Comment:
nit: Consistency with the rest of the Javadocs

```suggestion
     * Creates a PollerFlux instance that returns an error on subscription.
```